.github/workflows/build-and-test.yml

```diff
@@ -10,6 +10,10 @@ jobs:
   build:
     timeout-minutes: 10
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+        redis-version: [4, 5, 6]
```

Comment:
You don't use node, and this is overkill. Just remove this whole bit and put in a constant in place of `${{ matrix.redis-version }}`

Reply:
Yeah, I was just testing the example code